### ISSUE TITLE
Rendering Error With Pehkui

### ISSUE THREAD

**Issue by SophieBaybey** [ValkyrienSkies/Valkyrien-Skies-2#1012]
### Mod Name

Pehkui

### This issue occurs when only Valkyrien Skies, addons, and the mod I have specified are installed and no other mods

- [X] I have tested this issue and it occurs with only Valkyrien Skies, addons, and the mod I have specified

### Minecraft Version

1.20.x

### Mod Loader

Forge

### Issue description

Player renders incorrectly/behind the vehicle when sitting on a chair while Pehkui is installed. 
![image](https://github.com/user-attachments/assets/0ab10d67-8396-4bef-840b-f0dcf60d74bb)

### Issue reproduction

Install Pehkui, create VS vehicle, place a chair on the vehicle, sit on the chair.

### Logs

N/A

**Comment by thetheaplant** [comment]
Fixed in #927

**Comment by PriestOfFerns** [comment]
@thetheaplant  that's a PR for fixing collision with pekhui...

**Comment by SophieBaybey** [comment]
Yeah I didn't say anything because I wasn't sure if maybe fixing the movement bugs somehow inadvertently fixed this issue too, but I'm not entirely sure how they'd be related unless this bug was caused by collision with the chair somehow

**Comment by thetheaplant** [comment]
Yeah that appears to be my bad, reopening since this shouldn't be too hard to fix

**Comment by TechTastic** [comment]
Heres a video i took of this issue in action

https://streamable.com/00cwf8

**Comment by Slayer5934** [comment]
> Yeah that appears to be my bad, reopening since this shouldn't be too hard to fix

Hey, I love this mod and this is a blocking bug to get it into my modpack for people to enjoy, is it okay to do a bug bounty for $25-100?
 
I will be looking into trying to fix this myself as well but it will likely take me much longer x.x I've never had to fix an issue like this..

**Comment by JustOneMorePaul** [comment]
confirming this issue is still active as of the most recent beta 10 on 1.20.1

**Comment by jamesgreen26** [comment]
Fixed since #1412

**Comment by JustOneMorePaul** [comment]
Sadly, not on the version that still supports create 5.

**Comment by jamesgreen26** [comment]
Most mods have moved on to Create 6

@thetheaplant can we close this?

**Comment by thetheaplant** [comment]
yeap

**Comment by JustOneMorePaul** [comment]
Two of the mods that make create and VS significantly more fun to use together are not updated, so I think saying "most mods" are updated to create 6 is largely irrelevant. Trust me, I'd be on create 6 if I could be. I know mod updates take time, but writing create 5 off as unimportant is just odd to me.
